@xen-orchestra/lite/src/components/Tabs.tsx

```diff
@@ -0,0 +1,95 @@
+import Box from '@mui/material/Box'
+import React from 'react'
+import Tab from '@mui/material/Tab'
+import TabContext from '@mui/lab/TabContext'
+import TabList from '@mui/lab/TabList'
+import TabPanel from '@mui/lab/TabPanel'
+import Typography from '@mui/material/Typography'
+import { RouteComponentProps } from 'react-router-dom'
+import { withState } from 'reaclette'
+import { withRouter } from 'react-router'
+
+import IntlMessage from '../components/IntlMessage'
+
+interface ParentState {}
+
+interface State {
+  pathname: string
+}
+
+interface Tab {
+  component?: React.ReactNode
+  disabled?: boolean
+  label: React.ReactNode
+  pathname: string
+}
+
+// For compatibility with 'withRouter'
+interface Props extends RouteComponentProps {
+  indicatorColor?: 'primary' | 'secondary'
+  textColor?: 'inherit' | 'primary' | 'secondary'
+  // tabs= [
+  //   {
+  //      component: (<span>BAR</span>)
+  //      pathname: '/path',
+  //      label: (
+  //        <span>
+  //          <Icon icon='cloud' /> {labelA}
+  //        </span>
+  //      ),
+  //   },
+  // ]
+  tabs: Array<Tab>
+}
+
+interface ParentEffects {}
+
+interface Effects {
+  onChange: (event: React.SyntheticEvent, value: string) => void
+}
+
+interface Computed {}
+
+// TODO: improve view as done in the model(figma).
+const pageUnderConstruction = (
+  <div style={{ color: '#0085FF', textAlign: 'center' }}>
```

Comment:
Use `styled` div instead

Reply:
IMHO, we leave it as it is without changes. It can be a separate component in another file to make it look like the model on Figma. What do you think?